Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9-slim-buster as base
+
+WORKDIR /DevOps-Course-Starter
+COPY poetry.lock pyproject.toml /DevOps-Course-Starter/
```

Comment:
Minor, but note that since you've set this as your working directory you can use relative paths inside the container; `COPY poetry.lock pyproject.toml ./`